plugins/Kaleidoscope-Cycle/src/kaleidoscope/plugin/Cycle.h

```diff
@@ -20,7 +20,7 @@
 #include "kaleidoscope/Runtime.h"
 #include <Kaleidoscope-Ranges.h>
 
-#define Key_Cycle Key(kaleidoscope::ranges::CYCLE)
+constexpr Key Key_Cycle(kaleidoscope::ranges::CYCLE);
```

Comment:
Should this be:

constexpr Key Key_Cycle = Key(kaleidoscope::ranges::CYCLE);

?